acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/factory/PantheonNodeFactory.java

```diff
@@ -120,6 +121,24 @@ public PantheonNode createArchiveNodeWithP2pDisabled(final String name) throws I
             .build());
   }
 
+  public Node createFixedDifficultyMinerNode(final String name) throws IOException {
```

Comment:
`--network DEV` will be enough to get this now.

Reply:
or I could remove the method since fixed difficulty is being used in tests by default and the fixed difficulty AT has been deleted.